package.json

```diff
@@ -26,28 +23,44 @@
     "unit-test": "jest --selectProjects=Unit"
   },
   "dependencies": {
+    "@nestjs/cli": "^9.2.0",
     "@nestjs/common": "^9.3.9",
     "@nestjs/config": "^2.3.1",
     "@nestjs/core": "^9.3.9",
     "@nestjs/passport": "^9.0.3",
     "@nestjs/platform-express": "^9.3.9",
+    "@nestjs/schematics": "^9.0.4",
     "@nestjs/swagger": "^6.2.1",
     "@nestjs/terminus": "^9.2.1",
+    "@nestjs/testing": "^9.3.9",
     "@nestjs/typeorm": "^9.0.1",
+    "@tsconfig/node18": "^1.0.1",
+    "@types/chance": "^1.1.3",
+    "@types/compression": "^1.7.2",
+    "@types/express": "^4.17.17",
+    "@types/jest": "29.4.1",
+    "@types/node": "^18.15.0",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "compression": "^1.7.4",
     "date-fns": "^2.29.3",
     "dotenv": "^16.0.3",
     "express-basic-auth": "^1.2.1",
+    "husky": "^8.0.3",
+    "jest": "29.5.0",
+    "lint-staged": "^13.2.0",
     "mssql": "^9.1.1",
     "nestjs-pino": "^3.1.2",
     "passport": "^0.6.0",
     "passport-headerapikey": "^1.2.2",
     "pino-http": "^8.3.3",
     "pino-pretty": "^10.0.0",
     "reflect-metadata": "^0.1.13",
+    "rimraf": "^4.4.0",
     "rxjs": "^7.8.0",
+    "ts-jest": "29.0.5",
```

Comment:
Ts-jest and ts-loader shouldn't be needed in prod (i.e. as part of `dependency` section

Reply:
`ts-jest` unfortunately cannot be moved as it fails build process however `ts-loader` has been moved back to `devDependencies`.

Reply:
Something odd is happening then. Jest is a DevDependency. And ts-jest simply allows us to convert the `jest.confing.js` to `jest.config.ts`. it could be that we are copying files to the container that we shouldn't